sdk/core/Azure.Core.TestFramework/src/Instrumentation/DiagnosticScopeValidatingInterceptor.cs

```diff
@@ -209,9 +209,14 @@ internal static async ValueTask<T> ValidateDiagnosticScope<T>(Func<ValueTask<(T
                     }
                     else
                     {
-                        if (!diagnosticListener.Scopes.Any())
+                        // If ForwardsClientCallsAttribute is being used on the method, we don't know what the name of the scope should be because there could be many
```

Comment:
Why are Azure.Core scopes created?  Scopes are supposed to correspond to customer calls of public methods.  Or are these the inner HTTP scopes?

Reply:
Maybe we could defined "The Azure.Core scope?"

Reply:
Per https://github.com/Azure/azure-sdk-for-net/issues/27828#issuecomment-1115069467, perhaps we could call the "nested HTTP scopes"?

Reply:
> Why are Azure.Core scopes created? Scopes are supposed to correspond to customer calls of public methods. Or are these the inner HTTP scopes?

Azure.Core creates scopes for outgoing requests - https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/Pipeline/Internal/RequestActivityPolicy.cs#L57

Reply:
> , perhaps we could call the "nested HTTP scopes"?

I'm not sure what this comment is referring to.

Reply:
> there should be some scope created other than the Azure.Core scope.

It's a nit, but in the distributed tracing guidelines, I've heard these referred to as the "nested HTTP/REST spans", and never as Azure.Core scopes, see https://azure.github.io/azure-sdk/general_implementation.html#distributed-tracing

Reply:
Ah I see. That is a fair point.